components/automate-ui/src/app/pages/project/details/project-details.component.ts

```diff
@@ -89,6 +83,13 @@ export class ProjectDetailsComponent implements OnDestroy {
       });
   }
 
+  ngOnInit(): void {
+    this.projectForm = this.fb.group({
+      // Must stay in sync with error checks in project-details.component.html
```

Comment:
Is _now_ a good time to refactor this? 😉

Reply:
🤔 I'm not understanding your suggestion. That comment is reflecting upon an unavoidable consequence of how sometimes you have to refer to something in the code and in the template.

Reply:
Sorry, I've misread that. Skimming this, I thought it was another occurrance of the "must stay in sync with other thing" where we had agreed that it would be good to make it a constant.